### PR TITLE
Created primary_color_function

### PR DESCRIPTION
**Pull request by gulten-cicek**

### CODE REVIEW

File: gulten.py [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+    elif (primaryColor1 == "blue" and primaryColor2 == "yellow") or (primaryColor1 == "yellow" and primaryColor2 == "blue"):
+        print (primaryColor1 + " mixed with " + primaryColor2 + " is green")
+    else:
+        print ("One color, " + primaryColor1 + " or " + primaryColor2 + " isn't a primaryColor")
+
+
+primaryColor1 = validateUserString("Choose a color" ,["red","yellow","blue"],True ,3 ,"Please choose red, yellow, or blue")
+primaryColor2 = validateUserString("Choose a second color" ,["red","yellow","blue"] ,True , 3, "Please choose red, yellow, or blue")
+
+primary_color("primaryColor1, primaryColor2")
```
Comment (mlhaus) [inline gulten.py:20]: Don't put quotation marks around variable names. Only use quotation marks around strings. This should be `primary_color(primaryColor1, primaryColor2) `

**Review by mlhaus — CHANGES_REQUESTED** [review]
Remove the quotation marks around variable arguments when calling the function.

**Comment by gulten-cicek** [comment]
I will do it, thank you.
________________________________

**Comment by mlhaus** [comment]
You copied the tic-tac-toe code into your gulten.py file. That's not what I wanted you to do.

**Comment by gulten-cicek** [comment]
Do I have to send tic-tac-toe again?

________________________________

**Comment by mlhaus** [comment]
No, it was done as a class in the tic-tac-toe.py file. You need to send your exercise6 code for Chapter 7.

**Pull request closed without merging**